client/webserver/site/src/js/markets.js

```diff
@@ -534,6 +534,12 @@ export default class MarketsPage extends BasePage {
   async setMarket (host, base, quote) {
     const dex = app().user.exchanges[host]
     const page = this.page
+
+    // reset form inputs
+    page.lotField.value = 0
```

Comment:
Should just set this to `page.lotField.value = ''` too.

Reply:
If we set `lotField.value` to `''` instead of `0`, we can get an error in `lotChanged` if you delete the number:

![image](https://user-images.githubusercontent.com/9373513/156660733-39a5c3e6-0c4b-40be-b49f-afb451618cc7.png)

![image](https://user-images.githubusercontent.com/9373513/156660753-d9f977f8-d734-4601-b950-af9543a22f3a.png)

`lotField.value` seems to be numeric everywhere else.  Looking at `lotChanged` and `quantityChanged` functions, there it seems intentional to have the other fields as text with this one as numeric.

Reply:
The warnings seem to be independent of the resetting. I see them on the master also. If you clear the lot or the qty field, they appear in the console.
I've pushed a possible solution. To remain consistent, I set the `lotField` field to `''` if it's empty or negative until @buck54321 takes a look at it.

Reply:
I noticed a possible downside of my solution. It disallows writing a floating number to the quantity field. I don't know whether it is meant to be possible, but on master, it is already a bit tricky. I can only type floating quantity numbers on master by starting typing `.` or `,` and the cost of the warning.

<img width="621" alt="image" src="https://user-images.githubusercontent.com/52497040/156759045-34775e14-9b0c-4075-90e7-49fa63415bf9.png">

Reply:
To fix that warning, use

```Go
const lots = parseInt(page.lotField.value || 0)
```

Reply:
To fix that warning, use

```Go
const lots = parseInt(page.lotField.value || 0)
```